spring-cloud-task-samples/multiple-datasources/README.adoc

```diff
@@ -21,9 +23,25 @@ which one to be used for the Spring Cloud Task repository.
 $ mvn clean package
 ----
 
-== Run:
+== Execute sample using 2 embedded databases (default):
 
 [source,shell,indent=2]
 ----
 $ java -jar target/multiple-datasources-2.2.2.RELEASE.jar
 ----
+
+== Execute sample using 2 external databases:
+
+Using the `external` profile, users will be able to establish both the default `spring.datasource` data source and a `second.datasource` data source.
+For example:
+[source,shell,indent=2]
+----
+export spring_datasource_url=<your db url>
+export spring_datasource_username=<your db user name>
+export spring_datasource_password=<your db user password>
+export spring_datasource_driverClassName=org.mariadb.jdbc.Driver
+export second_datasource_url=jdbc:<your db url>
+export second_datasource_username=<your db user name>
+export second_datasource_password=<your db user password>
```

Comment:
I think `export second_datasource_driverClassName=<your second db driver class name>` is missing here. Do you agree?

Reply:
Ooops.   Thanks for catching